src/router.js

```diff
@@ -186,7 +186,7 @@ export class Router {
   * @param params The route params to be used to populate the route pattern.
   * @returns {string} A string containing the generated URL fragment.
   */
-  generate(name: string, params?: any): string {
+  generate(name: string, params?: any, absolute?: boolean = false): string {
```

Comment:
What do you think about using an options object for the third parameter, in case we ever need to add more options?